docs/development/visualize/development-embedding-visualizations.asciidoc

```diff
@@ -1,8 +1,8 @@
 [[development-embedding-visualizations]]
 === Embedding Visualizations
 
-There are two different angular directives you can use to insert a visualization in your page. 
```

Comment:
My editor automatically removed all the unnecessary trailing white-spaces in this file. Sorry for cluttering the PR with that.

Reply:
too bad ... else the stats of this PR would look much better :) (+73 -130 ... would change to +10 or sth i guess ? )